src/forms/yup.js

```diff
@@ -19,16 +19,27 @@ import * as yup from 'yup';
 
 const ARRAY_INDEX_REGEX = /\[([^)]+)\]/;
 
+const isCountrySelected = value => {
+  if (!value || value === '') {
+    return false;
+  }
+  return true;
+};
+
+yup.addMethod(yup.string, 'selected', function () {
+  return this.test(
+    'selected',
+    params => ({ key: 'form.validation_select_field_required', params }),
+    value => isCountrySelected(value)
```

Comment:
suggestion: Use something like `value => Boolean(value)`

Reply:
Thanks! Didn't know that was a built in function